### DISCUSSION TITLE
Ring Accessories Not Appearing in Home App

### DISCUSSION THREAD

**Discussion by nbanks93** [dgreif/ring#1213] [General]
Hello! I have been enjoying using Homebridge and this plugin for a few years to control my Ring alarm through Home. I also use some Unifi Protect cameras and a Nest Thermostat using other plugins.

After moving to a new computer, I was able to get the cameras and thermostat working again, but for some reason, the Ring plugin seems to work OK based on the logs, but then I see nothing in the Home app.

I have tried restarting Homebridge, deleting and setting up the Ring plugin again, refreshing the token, and deleting the Homebridge bridge in the Home app, but none of those attempts have solved my problem.

Any tips? Thanks in advance.

**Comment by dgreif** [comment]
Is it possible that the devices are there, but located in a room that you didn't expect? You mentioned the logs, do they say that the expected devices have been added?

**Reply by nbanks93** [reply]
A good thought! I can't tell which room they are being added to. I don't see them in the default room in Homebridge either. Do you know how I can check which room they are in from the logs?

**Reply by nbanks93** [reply]
I don't know what changed, but I messed with the plugin settings to only show one location and only one accessory type (the control pad) and that seemed to fix things. Not sure why!

**Reply by dgreif** [reply]
Hmm that's weird, but I'm glad it's working now!
